cw/cw305/simple_capture_traces.py

```diff
@@ -80,6 +77,21 @@ def run_capture(capture_cfg, ot, ktp):
 def plot_results(plot_cfg, project_name):
   """Plots traces from `project_name` using `plot_cfg` settings."""
   project = cw.open_project(project_name)
+
+  if len(project.waves) == 0:
+    print('Project contains no traces. Did the capture fail?')
+    return
+
+  # The ADC output is in the interval [-0.5, 0.5). Check that the recorded
+  # traces are within that range with some safety margin.
+  if not (np.all(np.greater(project.waves, -plot_cfg['amplitude_max'])) and
+          np.all(np.less(project.waves, plot_cfg['amplitude_max']))):
+    print('WARNING: Some traces have samples outside the range (' +
+          str(-plot_cfg['amplitude_max']) + ', ' +
+          str(plot_cfg['amplitude_max']) + ').')
+    print('\t The ADC has a max range of [-0.5, 0.5) and might saturate.')
+    print('\t It is recommended to reduce the scope gain (see device.py).')
```

Comment:
nit: Can you replace `\t` with spaces so that these lines are aligned with the text after `WARNING: `?

Reply:
Done.